session1/excercises/generics.ts

```diff
@@ -0,0 +1,11 @@
+class mobile{
+    constructor(public modelName: string,public price: number) {        
```

Comment:
Why do we need to provide the `public` access modifier inside the function? I saw this at some other places too. Please check for the same for every code from OOP onward.